src/preferences/constants.h

```diff
@@ -39,6 +39,13 @@ enum class MultiSamplingMode {
 };
 Q_ENUM_NS(MultiSamplingMode);
 
+enum class SoundHardwareTab {
+    Output,
+    Input,
+    Invalid
```

Comment:
do we really need `Invalid`?

Reply:
I added it to have a default `tab` argument for `showSoundHardwarePage`.

Thinking about it, that could also be std::optional.
Fixed.